forch/radius_query.py

```diff
@@ -1,8 +1,10 @@
 """Talks and listens to RADIUS. Takes a packet object as input"""
-from queue import Queue
 import logging
 import os
 
+from queue import Queue
```

Comment:
Why is this even here?  Is it used for anything?  If not, please remove.  And if it is... then I'm gonna want to take a deeper look at it :-)

Reply:
Oh yeah. That's an artifact of the sned queue I mentioned in the comment below. Let me remove it.

Reply:
done